### DISCUSSION TITLE
ESMPy on PyPI?

### DISCUSSION THREAD

**Discussion by tyfolino** [esmf-org/esmf#304] [Ideas / Requests]
It would be great from a development standpoint to not require conda to install ESMPy. Can ESMPy be put on PyPI so that it can be installed via pip? That way, packages that require ESMPy could be install solely via pip instead of having to create a conda environment.

I've used ESMPy for years. Thanks for developing it!

**Comment by danrosen25** [comment]
@anntsay
Can we add this to the backlog under CD (continuous deployment)?

**Comment by billsacks** [comment]
@tyfolino - Thank you for your input! We have this on our backlog (https://github.com/esmf-org/esmf/issues/256). Our understanding is that this will probably be non-trivial because of the dependencies of ESMPy, so I'm not sure when we'll realistically get to this, but we do recognize that there is substantial interest in this.
